deploy-aks/main.tf

```diff
@@ -0,0 +1,95 @@
+#########################################################
+### Defines Azure Resource Manager Terraform Provider ###
+#########################################################
+provider "azurerm" {
+    version = "2.9.0"
+    features {}
+}
+#########################################################
+##### Defines the Random string generator Provider ######
+#########################################################
+terraform {
+  required_providers {
+    random = {
+      source = "hashicorp/random"
+      version = "2.3.0"
+    }
+  }
+}
+#########################################################
+##### Defines the Random string generator Provider ######
+#########################################################
+resource "random_id" "prefix" {
+  byte_length = 5 
+}
+
+#########################################################
+######### Creates Azure Resource Group ##################
+#########################################################
+resource "azurerm_resource_group" "rg-cooldad-mvm" {
+  name = "rg-cooldad-mvm"
+  location = var.resource_location
+}
+
+#########################################################
+######### Creates Azure Virtual Network #################
+#########################################################
+resource "azurerm_virtual_network" "mvm-aks-vnet" {
+    name = "mvm-aks-vnet"
+    address_space = ["192.168.0.0/16"]
+    location = azurerm_resource_group.rg-cooldad-mvm.location
+    resource_group_name = azurerm_resource_group.rg-cooldad-mvm.name
+}
+
+
+#########################################################
+######### Creates Azure Subnet tied to VNET #############
+#########################################################
+resource "azurerm_subnet" "mvm-aks-subnet" {
+    name                 = "mvm-aks-subnet"
+    resource_group_name  = azurerm_resource_group.rg-cooldad-mvm.name
+    address_prefixes     = ["192.168.1.0/24"]
+    virtual_network_name = azurerm_virtual_network.mvm-aks-vnet.name
+}
+
+
+#########################################################
+######### Creates Azure Kubernetes Cluster ##############
+######### 1 Minecraft Container is Created ##############
+########## 2 CPU and 8 GB Memory Allocated ##############
+#########################################################
+resource "azurerm_kubernetes_cluster" "useast-minecraft-aks" {
```

Comment:
Do not hard code location, allow user to pick deployment location or inherit from resource group location.

Reply:
Definitely changed this, are you looking at the same commit as me?